pkg/corerp/README.md

```diff
@@ -43,14 +42,12 @@
     }'
     ```
 
-1. To get information about an environment, here is an example curl command:
+4. To get information about an environment, here is an example curl command:
 
     ```
     curl --location --request GET 'http://localhost:8080/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/radius-test-rg/providers/Applications.Core/environments/?api-version=2023-10-01-preview'
     ```
 
-1. You should also be able to see all changes in Cosmos DB
```

Comment:
Should we update this to point to etcd (and later to postgres once the work completes) instead of deleting?

Reply:
There's no straightforward way to do this with etcd. 

What do you think about making this part of the postgres work? @superbeeny and I talked about updating the dev instructions as part of that.

Reply:
that sounds good